cassandra-migration-spring-boot-starter/src/main/java/org/cognitor/cassandra/migration/spring/CassandraMigrationAutoConfiguration.java

```diff
@@ -29,34 +31,41 @@
 @ConditionalOnClass(CqlSession.class)
 public class CassandraMigrationAutoConfiguration {
     public static final String CQL_SESSION_BEAN_NAME = "cassandraMigrationCqlSession";
+    public static final String MIGRATION_TASK_BEAN_NAME = "migrationTask";
     private final CassandraMigrationConfigurationProperties properties;
 
     @Autowired
     public CassandraMigrationAutoConfiguration(CassandraMigrationConfigurationProperties properties) {
         this.properties = properties;
     }
 
-
-    @Bean(initMethod = "migrate")
+    @Bean(name = MIGRATION_TASK_BEAN_NAME, initMethod = "migrate")
     @ConditionalOnBean(value = CqlSession.class)
     @ConditionalOnMissingBean(MigrationTask.class)
     public MigrationTask migrationTask(@Qualifier(CQL_SESSION_BEAN_NAME) CqlSession cqlSession) {
         if (!properties.hasKeyspaceName()) {
-            throw new IllegalStateException("Please specify ['cassandra.migration.keyspace-name'] in" +
+            throw new IllegalStateException("Please specify ['cassandra.migration.keyspace.keyspace-name'] in" +
```

Comment:
I assume this is a leftover from an intermedtiate step because as far as I see `cassandra.migration.keyspace-name` is still the way to define this property, right?

Reply:
You're right ! I first implemented my own solution, then realized that this feature already exists in v3 branch so I just ported it to v4 for compatibility issue in case of upgrading the version.
Let me fix that !

Reply:
Fixed